service/src/main/java/org/cbioportal/genome_nexus/service/internal/VariantAnnotationSummaryServiceImpl.java

```diff
@@ -72,6 +77,10 @@ public VariantAnnotationSummaryServiceImpl(VariantAnnotationService hgvsVariantA
         this.variantClassificationResolver = variantClassificationResolver;
         this.variantTypeResolver = variantTypeResolver;
         this.exonResolver = exonResolver;
+
+        this.canonicalTranscriptResolver.setPredefinedCanonicalTranscriptIds(
+            this.ensemblService.getCanonicalTranscriptIdsBySource("genome_nexus")
```

Comment:
We should probably define enum or constant for the available isoform override sources.